### DISCUSSION TITLE
Possible to use Xbox Elite Controller paddles?

### DISCUSSION THREAD

**Discussion by KaittyL** [r57zone/DualShock4-emulator#65] [Q&A]
As simple as what I wrote above.

**Comment by r57zone** [comment]
Is there any problem with it?

**Reply by KaittyL** [reply]
Mine don't work personally and I haven't been able to find a way to get them working

**Comment by r57zone** [comment]
Try connecting with a wire

**Reply by KaittyL** [reply]
I don't have wireless on my PC so only way I've tried is with a wire

**Comment by r57zone** [comment]
try then to use x360ce along with this emulator

**Comment by r57zone** [comment]
https://www.x360ce.com
